libretroshare/src/services/p3banlist.cc

```diff
@@ -615,9 +615,10 @@ void p3BanList::getDhtInfo()
 
         addBanEntry(ownId, ad, RSBANLIST_ORIGIN_SELF, int_reason, time_stamp);
```

Comment:
That's not the error.
The problem is that 366 calls mBanRanges.find() without mutex protection. Same for the calls afterwards.
So The fix is to add a mutex call in the beginning of the function isAddressAccepted(), given that it doesn't call any other functions of the service. 
And for the record, adding {} here doesn't change anything.